src/Deserializers/LegacyPropertyDeserializer.php

```diff
@@ -95,10 +72,23 @@ private function getPropertyId() {
 	}
 
 	/**
-	 * @return Fingerprint
+	 * @param array $serialization
+	 * @return string
```

Comment:
The method throws that exception. There is no question about this, or is it? Tools complain about the missing documentation. Why introduce potential issues and confusion when the solution basically costs nothing? PHPStorm even automatically adds these lines. Why waste time on discussing this?